test/runtests.jl

```diff
@@ -8,11 +8,11 @@ using ITensors, Test
         "test_indexset.jl",
         "test_itensor.jl",
         "test_contract.jl",
+        "test_combiner.jl",
         "test_trg.jl",
         "test_ctmrg.jl",
         "test_dmrg.jl",
         "test_siteset.jl",
-        "test_phys_sitesets.jl",
```

Comment:
Did you intend to remove this line?

Reply:
no I don't know why it vanished :(. I'll put it back